temp2.ts

```diff
@@ -0,0 +1,7 @@
+if (false) {
+    console.log('nunca vai cair aqui')
+}
+
+window.setTimeout(() => {
+    console.log('sei lá o que eu to escrevendo')
+}, 300)
```

Comment:
ChatGPT Review:

    The code above will never print 'sei lá o que eu to escrevendo' to the console because the if statement will always evaluate to false. There is also a syntax error on line 6.